src/util/isEqual.ts

```diff
@@ -25,13 +25,24 @@ export default function isEqual(first: any, second: any, regexMatch = false): bo
   }
 
   if (firstIsArray) {
+    if(!regexMatch){
+     first.sort();
+     second.sort();
+    }
```

Comment:
please check my comment here:
 https://github.com/dushanpantic/grpc-server-mock/commit/0e5983309b07e6e3fb8cad4644422a3321268d5d#r64399531, and revert all changes to isEqual and isEqual.spec